test/tests/auxkernels/time_derivative_second_aux/test.i

```diff
@@ -0,0 +1,87 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  xmin = 0
+  xmax = 1
+  ymin = 0
+  ymax = 1
+  nx = 3
+  ny = 2
+[]
+
+[Variables]
+  [u]
+  []
+[]
+
+[Kernels]
+  [time]
+    type = TimeDerivative
+    variable = u
+  []
+  [reaction]
+    type = Reaction
+    variable = u
+  []
+  [diffusion]
+    type = Diffusion
+    variable = u
+  []
+[]
+
+[BCs]
+  [left]
+    type = NeumannBC
+    variable = u
+    value = 5
+    boundary = 'left'
+  []
+[]
+
+[AuxVariables]
+  [variable_derivative]
+    family = MONOMIAL
+    order = CONSTANT
+  []
+  inactive = 'variable_derivative_fv'
+  [variable_derivative_fv]
+    family = MONOMIAL
+    order = CONSTANT
+    fv = true
+  []
+[]
+
+[AuxKernels]
+  # Time derivative of a nonlinear variable
+  # Time derivative of an aux variable is not currently supported
+  # in the Auxiliary system (only in nonlinear)
```

Comment:
really? I believe you, but I know we attach time integrators to the auxiliary system so conceptually I feel like we should be able to. Do you get an error?

Reply:
this is pasted from the other test, where I wrote this most likely

I ll see what I can find

Reply:
ok seems to work. I tried two examples and got something reasonable with two integration schemes. 

I dont recall why I added this note